Userland/Applications/HexEditor/HexEditor.cpp

```diff
@@ -545,27 +560,19 @@ void HexEditor::paint_event(GUI::PaintEvent& event)
             is_current_line ? palette().ruler_active_text() : palette().ruler_inactive_text());
     }
 
-    for (int i = min_row; i < max_row; i++) {
-        for (int j = 0; j < bytes_per_row(); j++) {
+    for (size_t i = min_row; i < max_row; i++) {
+        for (size_t j = 0; j < bytes_per_row(); j++) {
             auto byte_position = (i * bytes_per_row()) + j;
-            if (byte_position >= static_cast<int>(m_buffer.size()))
+            if (byte_position >= m_buffer.size())
                 return;
 
             const bool edited_flag = m_tracked_changes.contains(byte_position);
 
-            auto highlight_flag = false;
-            if (m_selection_start > -1 && m_selection_end > -1) {
-                if (byte_position >= m_selection_start && byte_position < m_selection_end) {
-                    highlight_flag = true;
-                }
-                if (byte_position >= m_selection_end && byte_position < m_selection_start) {
-                    highlight_flag = true;
-                }
-            }
+            const bool highlight_flag = (byte_position >= m_selection_start && byte_position < m_selection_end) || (byte_position >= m_selection_end && byte_position < m_selection_start);
```

Comment:
This would look nicer in multiple lines

Reply:
Agreed and changed. However the tests don't like me much. Even though I had nothing to do with those. Promise.

Reply:
Tests are UE, I believe, so yeah, that's fine